doc/source/design.rst

```diff
@@ -335,10 +335,20 @@ Hooks are passed through Aura as an ADT of functions.
 Aesthetics
 ----------
 
+Size Information
+~~~~~~~~~~~~~~~~
+If ``--verbose-size`` is passed to Aura, the following information is
+displayed before installation from the official repositories.
+
+.. code-block::
+
+   Total download size: xx MiB
+   Net upgrade size   : xx MiB
```

Comment:
No, MiB is English (https://en.wikipedia.org/wiki/Mebibyte).
You can see it in the output of `ifconfig` for example, or gparted, and so on. Those binary prefixes are meant to replace the conflicting ones with the SI prefixes.

In French, it would be "Mio".